content/english/useless-box/_index.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Useless box: Silly gadget circuit"
+description: "A guide to build a 'useless box' to learn the basics of circuits and how a battery powers action."
+date: 2020-09-207T11:45:38-07:00
+prereq: "None"
+difficulty: "Beginner"
+download: ""
+draft: true
+icon: "fas fa-microchip"
+---
+
+# Can we make a box for Nuvi?
+
+Nuvi is in a bad mood, and it wants to be left alone. You have to make a box for it to have some privacy, but the box needs to be openable with a switch if you need to talk to it.
+Let’s make what’s called a “useless box”. This could also be amusement for little ones, or just a silly gadget that’s fun to play with and pass the time 😊
```

Comment:
```suggestion
Let’s make what’s called a “useless box”. This could also be amusement for little ones, or just a silly gadget that’s fun to play with and pass the time 😊.
```